src/main/java/com/amihaiemil/docker/RemoteDocker.java

```diff
@@ -45,24 +46,34 @@
     /**
      * Remote Docker engine. API version is 1.35 by default.
      * @param uri Remote Docker URI.
-     * @param certs Path to the folder containing these 3:
-     *  CA certificate (ca.pem), client certificate (cert.pem)
-     *  and client key (key.pem).
+     * @param keys Path to the keystore.
+     * @param trust Path to the truststore.
+     * @param storePwd Password for the keystore.
+     * @param keyPwd Passphrase for the key.
      */
-    public RemoteDocker(final URI uri, final Path certs) {
-        this(uri, "v1.35", certs);
+    public RemoteDocker(
```

Comment:
@llorllale I see... so you want to let the user provide the KeyStore and TrustStore from disk. I was initially thinking we can read the certificate' contents and build those objects ourselves, in-memory (same as Spotify is doing in their client).

But maybe your option is better: whoever uses a remote Docker will most likely want to register those certificates in proper key/trust stores, together with other keys. We would just need to provide proper documentation so the user knows what to do.